### DISCUSSION TITLE
Openai module support

### DISCUSSION THREAD

**Discussion by lethargistic** [pyscript/pyscript#1617] [Proposals]
Official opeanai API module is currently not supported so you have to go around and send an xhr directly, and that disallows the use of ChatGPT 3.5 and 4 anyways so it's limited.

Would be great to have officially supported it since that'll finally allow using the Python version of the API for websites without a server.

**Comment by JeffersGlass** [comment]
This is, I think, more of an upstream issue with Pyodide. It looks like they have an [open issue about adding openai](https://github.com/pyodide/pyodide/issues/3588) - I'd recommend adding your comments there as well, since that helps their core devs know what packages are most desired.

**Reply by lethargistic** [reply]
I will, thank you for informing.
